### DISCUSSION TITLE
Automatic popup behavior of tag dropdown in 5.1.23

### DISCUSSION THREAD

**Discussion by hoelzro** [TiddlyWiki/TiddlyWiki5#5405] [General]
Hello!

I noticed that as of TiddlyWiki 5.1.23, the tag dropdown popup automatically re-opens upon selecting a tag with the mouse.  If I understand correctly, this resulted from some of the keyboard-driven enhancements as part of the release.

I'm all for keyboard-driven improvements, but I (personally) find this somewhat jarring!  I might be the only user put off by this behavior, so I wanted to start a discussion to see if there are others who feel this way, and/or to see if there's a more desirable UI pattern the dropdown could employ.  Two ideas I've had around this are:

  * Only re-open the popup if the user selected a tag via the keyboard
  * (Probably better) Only open the tag dropdown after the user has entered a minimum number of characters, much like how the search sidebar works

Please chime in if you have ideas on how to improve this part of the UI!

**Comment by pmario** [comment]
You can hit the TAB key **2** times or select the text area with the mouse and the dropdown goes away.

**Reply by hoelzro** [reply]
Right - that much I had figured out :sweat_smile:.  I was mostly hoping for a solution other than "do this thing you didn't have to before" that still preserves the new functionality!

**Comment by hoelzro** [comment]
I was poking around in the code for the dropdown and stumbled upon `$:/config/Tags/MinLength` - this solves my issue perfectly!

**Reply by Jermolene** [reply]
Good tip, thanks @hoelzro
